core/templates/pages/story-editor-page/editor-tab/story-editor.directive.html

```diff
@@ -228,6 +231,17 @@ <h3 class="story-card-header">Story Card</h3>
 </div>
 
 <style>
+  .story-length-validation-error {
+    align-items: center;
+    background-color: #f8d7da;
+    border: 0.0625rem solid #dc3545;
+    border-radius: 0.25rem;
+    color: #dc3545;
+    display: flex;
+    justify-content: center;
+    padding: 0.25rem;
+  }
+
```

Comment:
Do we need this special CSS? Or can we take the one that you used in other places in your previous PR?

Reply:
Actually, I took a look at this PR [here](https://github.com/oppia/oppia/pull/14035) where this CSS was used and it looked nice and srijan was recommending to put it in the main CSS, that's why I did it but please do let me know if I should change.
Thanks

Reply:
Sounds good -- no changes needed!